pkg/ceph/client/image.go

```diff
@@ -139,6 +139,10 @@ func UnMapImage(context *clusterd.Context, imageName, poolName, clusterName, key
 		"--conf=/dev/null", // no config file needed because we are passing all required config as arguments
 	}
 
+	if force {
```

Comment:
Just making sure, if the volume is still mounted, using `-o force` will not get you a `resource busy` error right?

Reply:
i have not seen any errors returned yet when i've forcefully unmapped a volume out from underneath a pod using this `-o force` argument.  There may be conditions, such as when there's in flight I/O that is not responsive, that could prevent the unmapping from completing, and we will timeout and return.

In the future, there will be `-o full-force` that will REALLY stop everything, but we'll have to wait until that has been implemented and available in the linux kernel.  more details here: https://github.com/rook/rook/issues/376#issuecomment-319019885